requirements-dev.txt

```diff
@@ -1,17 +1,17 @@
-black==20.8b1
+black
 flake8
 freezegun
-isort==5.7.0
+isort
 moto==1.3.16
 mypy==0.790
 numpydoc==1.1.0
 pandas
 pdbpp
 pyarrow
 pyspark==3.1.1
-pytest==5.2.2
+pytest
 recommonmark
 sphinx
 sphinx_rtd_theme
 tox
-vcrpy==4.1.1
+vcrpy
```

Comment:
Could we keep the pins here as they were? Happy to bump these versions in follow up PRs but if for example we leave `black` unpinned, then devs on different versions could produce different formatting outputs.